featuretools/computational_backends/feature_set_calculator.py

```diff
@@ -898,3 +923,20 @@ def strip_values_if_series(values):
     if isinstance(values, pd.Series):
         values = values.values
     return values
+
+
+def is_agg_type_in_primitive_or_feature(f):
+    is_agg_type = False
+    if hasattr(f.primitive, "is_agg_type"):
+        return f.primitive.is_agg_type
+
+    if hasattr(f, "is_agg_type"):
+        return f.is_agg_type
+
+    if isinstance(f, AggregationFeature):
+        is_agg_type = "agg_type" in f.primitive.get_function.__code__.co_varnames
+        f.primitive.is_agg_type = is_agg_type
```

Comment:
@rwedge Let me know if this matches what you were thinking of.

Reply:
I think if we made this a class method for `AggregationPrimitive` we would only need to check once per primitive class because we'd updated the attribute for the class